playwright.config.ts

```
@@ -6,20 +6,111 @@
+        viewport: { width: 390, height: 844 }, // Mobile viewport on desktop for PWA testing
+        contextOptions: {
+          // PWA-specific options
+          permissions: ['notifications', 'geolocation'],
+        },
+      },
+    },
+
+    // Slow network testing for offline functionality
+    {
+      name: 'Mobile Slow 3G',
+      use: { 
```

ESLint

[error] Delete `·`

(prettier/prettier)